Updates to data sender and pose calculator from robot testing

michelllepan opened the pull request.

michelle-pan committed:
Add option to display image when batch testing

michelle-pan committed:
Take absolute value of angle values before calculating error

michelle-pan committed:
Add test photos with actual power port target

tnarayanan committed:
Fix constant angle error

michelle-pan committed:
Fix parsing of distance value from filename in batch_test.py

michelle-pan committed:
Rotate translation vector and update distance constant

tnarayanan committed:
fix angle constant

tnarayanan committed:
Remove transformations code from data sender

michelle-pan committed:
Add test photos from camera on robot

AG6GR committed:
Changes from Feb 18 meeting, tested on jetson

michelle-pan committed:
Return None if only a partial target is detected

michelle-pan committed:
Remove absolute value from output analysis

michelle-pan committed:
Display image name as window title

michelle-pan committed:
Make imporvements to data sender

michelle-pan committed:
Fix style

michelle-pan committed:
Fix partial target detection

michelle-pan committed:
Add missing import

AG6GR committed:
Fix merge conflict

AG6GR commented on pose_calculator.py:
```diff
@@ -143,30 +149,54 @@ def get_values(self, units="m", display=True):
-        # rx, ry, rz = self.get_euler_from_rodrigues(rmat)
-        r_euler = R.from_rotvec(rvec.flatten()).as_euler('xyz', degrees=True)
-        rx, ry, rz = r_euler[0], r_euler[1], r_euler[2]
+        rx, ry, rz = self.get_euler_from_rodrigues(rmat)
```
Is there a reason we swaped back to using the rodrigues matrix? If so we should remove the scipy import

AG6GR reviewed (approved):
Please check that this code still works. Had to repush to clear the extra merge commit that was generated.

michelle-pan committed:
Clean up code

1. Remove use of scipy in PoseCalculator
2. Add camera tilt constant in generators
3. Display image name in frame when using batch test
